### DISCUSSION TITLE
Using Answer To Select(ATS) string to detect the HCE

### DISCUSSION THREAD

**Discussion by mohaomer12** [nfcgate/nfcgate#154] [Q&A] [answered]
Hello,
It is my understanding that when employing clone mode, NFCGate also duplicates the Answer to Select (ATS), which embodies the characteristics of the original NFC card. Consequently, I am contemplating whether this feature could be utilized as a method to detect Host Card Emulation (HCE), given that the replicated properties may not align with the  HCE properties.

**Comment by kleest** [comment, marked as answer]
Yes, this could theoretically be possible if the cloned or relayed card has different characteristics. This is not limited to clone mode. 
The Android NFC stack sets certain values in the ATS (see [Android documentation](https://developer.android.com/develop/connectivity/nfc/hce#iso-dep-act)). NFCGate does not change or manipulate these values.
